geoserverexplorer/test/testplugin.py

```diff
@@ -52,4 +54,5 @@ def runAllUnitTests():
     suite.addTest(deleteSuite())
     suite.addTest(dragdropSuite())
     suite.addTest(guiSuite())
+    suite.addTest(symbologySuite())
```

Comment:
@ddespres we are changing the docker testing images to newer builds with GDAL 2 and based on xenial, the new images should be ready in ~ 60 minutes.
Please re-run the tests later.